framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -91,6 +91,7 @@ namespace bpl {
 #define DEFAULT_BAND_STEERING 1
 #define DEFAULT_DFS_REENTRY 1
 #define DEFAULT_CLIENT_ROAMING 1
```

Comment:
Why not convert these to `constexpr` while you're at it?

Reply:
replaced `default values defines` with `constexpr int`